samples/tab-deeplink/csharp/DeepLinkBot/DeepLinkHelperChannel.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Text;
+using System.Web;
+
+namespace Microsoft.BotBuilderSamples.Bots
+{
+    public class DeepLinkHelperChannel
+    {
+        public static string channelID = DeepLinkBot.channelID;
+
+        static string task1Values =>  task1Json();
+        static string task1Json()
+        {
```

Comment:
Can we create function which takes parameters and return deeplink string? 

public string GetDeeplinkToTab(string appId, string entityId, string subEntitiyId, string channelId)

Reply:
Created method but we need to send subentity ID for every deeplink on click adaptive card button